### PR TITLE
 Add pre-release version changelog to version list 

### PR DESCRIPTION
**Pull request by CyberShadow**
Depends on (and includes) #1766 (due to an unavoidable merge conflict otherwise).

### CODE REVIEW

File: changelog/changelog.ddoc [hunk 1]
```diff
@@ -64,8 +64,10 @@ PHOBOSPR = $(PULL_REQUEST phobos,$1)
 _=
 
 CHANGELOG_VERSION = $(LI <a id="$1" href="$1.html">$1</a><span class="hide-from-nav"> ($2, $3)</span>)
+CHANGELOG_VERSION_PRE = $(LI <a id="$1" href="$1_pre.html">$1</a><span class="hide-from-nav"> (not yet released)</span>)
 _=BEGIN_GENERATED_CHANGELOG_VERSIONS
 CHANGELOG_VERSIONS =
+    $(CHANGELOG_VERSION_PRE 2.075.0, not yet released)
```
Comment (wilzbach) [inline changelog/changelog.ddoc]: "not yet released" is redundant here - you only use `$1`  in `CHANGELOG_VERSION_PRE`

Comment (CyberShadow) [inline changelog/changelog.ddoc]: This part is generated by the script, so excluding _PRE from having its release date extracted would be more complicated.

I could make _PRE use $2 I guess.

**Comment by CyberShadow** [comment]
Would be good to build the changelog from the correct branch somehow, I guess.

**Comment by wilzbach** [comment]
Yeah, but how would the script know that the "stable-branch-off" has
happened?

**Comment by wilzbach** [comment]
(rebased to rekick DAutoTest as the changelog has been manually added now)

**Comment by CyberShadow** [comment]
SIgh, what happened with the "not yet released" text on http://dlang.org/changelog/2.075.0_pre.html?

@MartinNowak Could we please keep "not yet released" on that page up until the release date? We already had questions about it when someone found the page on Google what was up with the future release date.

**Pull request merged**